tests/SmokeTests/ProjectWebTestCase.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\SmokeTests;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+abstract class ProjectWebTestCase extends WebTestCase
+{
+    public static function createClient(array $options = [], array $server = [])
+    {
+        $client = parent::createClient($options, $server);
+
+        $server['HTTP_HOST'] = static::getContainer()->getParameter('app.testing.site_domain');
```

Comment:
Where does this parameter come from?

Reply:
Finished the implementation in c892c41e
Deployer now runs multiple times, for all configured domains:
```[17:04:17] iherak@azathoth: media-site [NGSTACK-492-smoke-tests] $ dep app:test:phpunit prod -vvv
➤ Executing task app:test:phpunit
[localhost] > TEST_DOMAIN=media-site.dev.php81.ez /usr/bin/env php8.1 vendor/bin/phpunit
[localhost] < PHPUnit 9.6.18 by Sebastian Bergmann and contributors.
[localhost] < Testing
[localhost] < Test page: http://media-site.dev.php81.ez/bold_eng/careers?nekisufiks
[localhost] < Test page: http://media-site.dev.php81.ez/fh_eng/recipes/greek-quinoa-salad
[localhost] < Test page: http://media-site.dev.php81.ez/fh_eng/fitness/effects-of-cadence-training-on-running-biomechanics-and-efficiency
[localhost] < .
[localhost] < .
[localhost] < .                                                                 3 / 3 (100%)
[localhost] < Time: 00:02.830, Memory: 237.23 MB
[localhost] < OK (3 tests, 9 assertions)
[localhost] < Remaining self deprecation notices (1)
[localhost] < Remaining indirect deprecation notices (437)
[localhost] < Other deprecation notices (66)
[localhost] > TEST_DOMAIN=media-site.dev.php81.ez/bold_eng /usr/bin/env php8.1 vendor/bin/phpunit
[localhost] < PHPUnit 9.6.18 by Sebastian Bergmann and contributors.
[localhost] < Testing
[localhost] < Test page: http://media-site.dev.php81.ez/bold_eng/#
[localhost] < Test page: http://media-site.dev.php81.ez/bold_eng/services
[localhost] < Test page: http://media-site.dev.php81.ez/bold_eng/services
[localhost] < .
[localhost] < .
[localhost] < .                                                                 3 / 3 (100%)
[localhost] < Time: 00:01.986, Memory: 209.23 MB
[localhost] < OK (3 tests, 9 assertions)
[localhost] < Remaining self deprecation notices (1)
[localhost] < Remaining indirect deprecation notices (418)
[localhost] < Other deprecation notices (66)
• done on [prod]
✔ Ok [5s 33ms]
```